calico-vpp-agent/vpp_manager_watchers/pool_sync.go

```diff
@@ -34,11 +34,13 @@ import (
 
 type PoolWatcher struct {
 	RouteWatcher *RouteWatcher
-	params       *config.VppManagerParams
-	conf         *config.LinuxInterfaceState
 
 	watcher              watch.Interface
 	currentWatchRevision string
+	UserSpecifiedMtu int
+	Mtu int
```

Comment:
This line tells me that you are not using gofmt (or even better goimports) tool. It is quite common to format the go code with it as it is the code format standard that has support in standard go tooling. This repo doesn't have it as requirement for contributions so no need to apply it. However, for someone using it (i.e. in IDE), it could create unnecessary ghost changes (just formatting) in commits that can be disruptive (for committer if he/she wants to get rid of it before commit or for reviewer if committer doesn't care about formatting ghost changes)

Reply:
thanks, just forgot to run the formatter ;)